core/types/log.go

```diff
@@ -69,14 +69,10 @@ type rlpLog struct {
 }
 
 type rlpStorageLog struct {
-	Address     common.Address
-	Topics      []common.Hash
-	Data        []byte
-	BlockNumber uint64
-	TxHash      common.Hash
-	TxIndex     uint
-	BlockHash   common.Hash
-	Index       uint
+	Address common.Address
+	Topics  []common.Hash
+	Data    []byte
+	Index   uint
```

Comment:
Do we need the index? Seems we can also calculate that with a simple accumulator across all the logs in a receipt.